cmd/zb/perf.go

```diff
@@ -386,6 +386,8 @@ func PushMonolithStreamed(workdir, url, auth, trepo string, requests int,
 	return nil
 }
 
+// nolint: godox
+// TODO refactor -  fix for maintidx linter.
```

Comment:
remove this nolint since we have disabled maintidx linter?

Reply:
godox is a separate linter that triggers separately - Line contains TODO/BUG/FIXME: "TODO refactor -  fix for maintidx linter..." (godox) .
without nolint, the above error will pop-up.

Reply:
actually, remove this TODO comment entirely since maintidx is disabled for now. we can revisit if/when it is enabled and complains.

Reply:
removed.